components/ProgressBar.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useContext } from "react";
+import { TodoContext } from "../store/store-todo";
```

Comment:
Look this: [https://stackoverflow.com/questions/43281741/how-to-use-paths-in-tsconfig-json](https://stackoverflow.com/questions/43281741/how-to-use-paths-in-tsconfig-json)